content/en/docs/setup/_index.md

```diff
@@ -4,11 +4,97 @@ reviewers:
 - erictune
 - mikedanese
 no_issue: true
-weight: 30
 title: Setup
 main_menu: true
+weight: 30
+content_template: templates/concept
 ---
+{% raw  %}
+
+{{% capture overview %}}
+
+You can run Kubernetes almost anywhere, from your laptop to VMs on a cloud
+provider to a rack of bare metal servers. Deciding where to run Kubernetes
+depends on what resources you have available and how much flexibility you need.
+
+Set up a fully-managed cluster by running a single command or craft your own
```

Comment:
IMHO, it might be better structured:

> Use this page to find the type of solution that best fits your needs.
>
> Deciding where to run Kubernetes depends on what resources you have available and how much flexibility you need. You can run Kubernetes almost anywhere, from your laptop to VMs on a cloud provider to a rack of bare metal servers. You can also set up a fully-managed cluster by running a single command or craft your own customized cluster on your bare metal servers.

That way you state the purpose of the page upfront. Then, in the second paragraph, you lead w/ specifying the two dimensions: resources available and flexibility. The following sentences then describe the range for each of those dimensions.

Reply:
Ah, good call!